### DISCUSSION TITLE
How to remove the magnet between node and grid when moving node?

### DISCUSSION THREAD

**Discussion by No54** [antvis/X6#2433] [Q&A] [answered]
How to remove the magnet between node and grid when moving node? ( grids are hidden )
Thanks！

请问如何去掉拖动节点时的磁吸？轴网已经隐藏了，就是拖动的时候，不流畅，平移整个图的时候就很流畅
谢谢！

**Comment by NewByVector** [comment, marked as answer]
拖动的时候是根据 gridSize 来平移节点的，可以将 gridSize 设置小一点。

**Reply by No54** [reply]
谢谢！
